addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -292,7 +292,22 @@ def migration_invoice_moves(env):
         )
         if not env.cr.fetchone():
             break  # exit condition not having more duplicates
-    # 2st: exclude from invoice_tab the grouped ones, and create a new separated ones
+    # Checking the missing lines to import
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_invoice_line ail
+        SET aml_matched = FALSE
```

Comment:
I don't see this assignation, as there shouldn't be any incorrect put as True. Take into account that this assignation can take a lot of minutes on a very populated table. I think the problem is that the default is NULL, not False/True. Then you should use `WHERE ail.aml_matched is not TRUE` as we have done in other places.

Reply:
`WHERE ail.aml_matched IS DISTINCT FROM TRUE` is the correct one that excludes both null and False.

Reply:
When we migrated, something didn't work if we didn't apply this change :disappointed: 

I can remove it if you prefer and set the IS DISTINCT FROM TRUE

Reply:
Yeah, let's do this, and merge. If then you still face the same problem, we can make another PR :)